app/Resources/static/themes/material/css/main.css

```diff
@@ -596,6 +645,47 @@ a.original {
   height: auto;
 }
 
+#article article a {
+  border-bottom: 1px dotted #03a9f4;
+  text-decoration: none;
+}
+
+#article article a:hover {
+  border-bottom-style: solid;
+}
+
+#article article ul {
+  padding-left: 30px;
+}
+
+#article article ul,
+#article article ul li {
+  list-style-type: disc;
+}
+
+#article article blockquote {
+  font-style: italic;
+}
+
+#article article strong {
+  font-weight: bold;
+}
+
+#article article pre {
+  -moz-box-sizing: border-box;
```

Comment:
No need for this `-moz` prefix since Firefox 29, I guess we don't need it anymore.